web/src/app/results/[data]/TextInfo.tsx

```diff
@@ -1,24 +1,42 @@
 import { EmotionResults } from "./page";
 
 export const TextInfo = ({ results }: { results: EmotionResults }) => {
+  const emotionColors: { [key: string]: string } = {
+    Neutral: "text-blue-300",
+    Anger: "text-red-600",
+    Disgust: "text-green-800",
+    Fear: "text-purple-600",
+    Happiness: "text-green-600",
+    Sadness: "text-gray-600",
+    Surprise: "text-yellow-600",
+  };
+
+  const emotionDescriptions: { [key: string]: string } = {
+    Neutral: "neutral",
+    Anger: "angry",
+    Disgust: "disgusted",
+    Fear: "fearful",
+    Happiness: "happy",
+    Sadness: "sad",
+    Surprise: "surprised",
+  };
+
+  // Sort emotions by total_seconds in descending order
+  const sortedEmotions = results.emotions.sort((a, b) => b.total_seconds - a.total_seconds);
+
   return (
     <>
       <p className="font-bold">
-        Your video has {results.total_frames} frames long
-      </p>
-      <p className="font-bold">
-        for{" "}
-        {results.emotions.find((e) => e.label === "Happiness")?.total_frames}{" "}
-        frames you looked <span className="text-green-600">happy</span>
-      </p>
-      <p className="font-bold">
-        for {results.emotions.find((e) => e.label === "Neutral")?.total_frames}{" "}
-        frames you looked <span className="text-blue-300">neutral</span>
-      </p>
-      <p className="font-bold">
-        for {results.emotions.find((e) => e.label === "Surprise")?.total_frames}{" "}
-        frames you looked <span className="text-yellow-600">surprised</span>
+        Your video is {results.total_seconds} seconds long
       </p>
+      {sortedEmotions.map((emotion) => (
+        <p key={emotion.label} className="font-bold">
+          for {emotion.total_seconds} seconds you looked{" "}
```

Comment:
@nicolaslongo , @cotarelorodrigo qué opinan de cambiar esta línea por algo como esto:

`for {emotion.total_seconds === 0 ? 1 : emotion.total_seconds} seconds you looked{" "}`

No es muy elegante, pero actualmente tenemos casos donde dice "for 0 seconds you looked sad" por ejemplo, lo cual no queda muy bien.
Si la emoción llegó hasta acá es porque al menos una secuencia tiene, por eso digo de ponerle 1 segundo.

Reply:
me parece más elegante que dejar "for 0 seconds you looked sad" lo cual implica la no información

Reply:
Este es un muy buen ejemplo
![image](https://github.com/fernecuba/FERNEC/assets/23227386/44f6c3c5-8a6e-48b4-893c-58bd3cdf3072)
Prefiero que la suma de 21, o filtrar el mensaje de 0 seconds

Reply:
Buen ejemplo, no había encontrado uno donde se diera esto